.gitignore

```diff
@@ -0,0 +1,7 @@
+dbsync-central/dbsync.env
```

Comment:
Why are you adding these files to .gitignore? I think the question should be why do you have them in the repo anyways?

Reply:
This file is refenced in docker-compose file and contain some sensitive information.

Reply:
Who added it? If the file is marked as ignored, you should just not include it in the repo anyways

Reply:
Already removed all sensitive files from repo